glean-core/src/histogram.rs

```diff
@@ -257,4 +257,17 @@ mod test {
         assert_eq!(1, hist.values[57]); // bucket_ranges[57] = 964
         assert_eq!(1, hist.values[80]); // bucket_ranges[80] = 9262
     }
+
+    #[test]
+    fn accumulate_large_numbers() {
+        let mut hist = Histogram::exponential(1, 500, 10);
```

Comment:
*Maybe* test large numbers for min/max here, too.  I say *maybe* because it seems a little insane to support that -- better to keep these build-time constants as reasonable.

Reply:
Yeah, I don't think we need to do that. We should make Glean just not accept huge values in the first place (heck, we don't even allow modification of that yet :D)
This test in particular is more of a regression test, so I'd like to keep it to the one thing only.

We can think about testing the boundaries of the histogram implementation itself, but then again we do know that above 53 bits we lose precision.... ¯\_(ツ)_/¯